pkg/util/informermanager/single-cluster-manager.go

```diff
@@ -39,12 +47,15 @@ func NewSingleClusterInformerManager(client dynamic.Interface, defaultResync tim
 	return &singleClusterInformerManagerImpl{
 		informerFactory: dynamicinformer.NewDynamicSharedInformerFactory(client, defaultResync),
 		handlers:        make(map[schema.GroupVersionResource][]cache.ResourceEventHandler),
+		syncedInformers: make(map[schema.GroupVersionResource]sets.Empty),
```

Comment:
Why use a `struct{}` from the `set` package? 
I think the semantic here is not a `set`, even though `sets.Empty` essentially is a `struct{}`.

Reply:
Just consider `syncedInformers` as a set. `struct{}` is OK, let me modify it.